.github/workflows/build_and_test_scala213_daily.yml

```diff
@@ -0,0 +1,115 @@
+name: Build and test Scala 2.13 daily
+
+on:
+  schedule:
+  - cron: '0 4 * * *'
```

Comment:
The daily snapshot publish job is scheduled on `0 0 * * *`. So, I used 4AM (GMT) to avoid resource competition.